site/content/en/docs/getting_started.md

```diff
@@ -15,21 +15,68 @@ To read about the design concept and features of Datumaro, go to the [design sec
 - Python (3.7+)
 - Optional: OpenVINO, TensorFlow, PyTorch, MxNet, Caffe, Accuracy Checker, Git
 
-Optionally, create a virtual environment:
+### Installation steps
+
+Optionally, set up a virtual environment:
 
 ``` bash
 python -m pip install virtualenv
 python -m virtualenv venv
 . venv/bin/activate
 ```
 
-Install Datumaro package:
-
+Install:
 ``` bash
+# From PyPI:
 pip install datumaro[default]
 ```
+``` bash
+# From the GitHub repository:
+pip install 'git+https://github.com/openvinotoolkit/datumaro[default]'
+```
+
+Read more about choosing between `datumaro` and `datumaro[default]`
+[here](#core-install).
```

Comment:
Is it possible to link by [here](#customizing-installation) without introducing `<a id="core-install"></a>`?